tests/test_datasets/test_dataset_functions.py

```diff
@@ -876,6 +876,94 @@ def test_get_online_dataset_format(self):
             "The format of the ARFF files is different",
         )
 
+    def test_create_dataset_default_target_attribute_validation(self):
```

Comment:
I know we haven't used this functionality before in this project, but would you be able to refactor these similar tests using `pytest.mark.parametrize`? It allows you to easily change only a few variables with otherwise same test setup.